frontend/integration-tests/tests/kubevirt/vm.overview.scenario.ts

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable no-undef */
+import { browser } from 'protractor';
+import { execSync } from 'child_process';
+import { OrderedMap } from 'immutable';
+
+import { appHost, testName } from '../../protractor.conf';
+import { isLoaded } from '../../views/crud.view';
+import { basicVmConfig, cloudInitCustomScriptConfig } from './mocks';
+import * as vmView from '../../views/kubevirt/virtualMachine.view';
+import { fillInput, removeLeakedResources, exposeService, selectDropdownOption } from './utils';
+import Wizard from './models/wizard';
+import { VirtualMachine } from './models/virtualMachine';
+
+describe('Test VM overview page', () => {
+  const leakedResources = new Set<string>();
+  const wizard = new Wizard();
+  const vmName = `vm-${testName}`;
+  const vm = new VirtualMachine(vmName, testName);
+
+  beforeAll(async() => {
+    const provisionSource = {method: 'Container', source: basicVmConfig.sourceContainer};
+
+    await browser.get(`${appHost}/k8s/ns/${testName}/virtualmachines`);
+    await isLoaded();
+    await wizard.openWizard();
+
+    // Basic Settings for VM
+    await wizard.fillName(vmName);
+    await wizard.fillDescription(testName);
+    await wizard.selectProvisionSource(provisionSource);
+    await wizard.selectFlavor(basicVmConfig.flavor);
+    await wizard.selectOperatingSystem(basicVmConfig.operatingSystem);
+    await wizard.selectWorkloadProfile(basicVmConfig.workloadProfile);
+    await wizard.useCloudInit(cloudInitCustomScriptConfig);
+    await wizard.next();
+
+    // Networking
+    await wizard.next();
+    // Storage
+    await wizard.next();
+    // Create VM
+    await wizard.next();
+
+    leakedResources.add(JSON.stringify({name: vmName, namespace: testName, kind: 'vm'}));
+  });
+
+  afterAll(async() => {
+    removeLeakedResources(leakedResources);
+  });
+
+  beforeEach(async() => {
+    await vm.navigateToTab(vmView.overviewTab);
+    await isLoaded();
+  });
+
+  it('Check VM details in overview when vm is off', async() => {
+
+    // Non empty fields when vm is off
+    expect(vmView.vmDetailNameID(testName, vmName).getText()).toEqual(vmName);
```

Comment:
you could also create a VmDetail class which takes namespace and name in a constructor and calls these methods for you. So you don't have to fill in `testName` & `vmName` every time

Reply:
This would be great, I have been thinking about having detailView and listView classes that would be properties of VirtualMachine and would provide this common functionality not just for VMs but for example templates as well.
However, I'm fine with implementing this in a follow up PR as it would be a rather big change.
@suomiy What do you think?

Reply:
np, we can postpone this to a followup